doc/api/n-api.md

```diff
@@ -2495,8 +2495,8 @@ of the ECMAScript Language Specification.
 #### `node_api_symbol_for`
 
 <!-- YAML
-added: REPLACEME
-napiVersion: REPLACEME
+added: v17.5.0
+napiVersion: v17.5.0
```

Comment:
@nodejs/node-api is that correct?

Reply:
Based on how other node-api API's exited experimental status (e.g. https://github.com/nodejs/node/commit/d15475578abfcc7e0a420719b8d55a6b3ab4f198) perhaps https://github.com/nodejs/node/pull/41329 should have omitted `napiVersion` metadata?

Reply:
in case that's a problem, is there a chance this is something we can patch later? I'm planning to push the release in the next few hours

Reply:
I think this is purely informational so could be patched later? cc @nodejs/node-api

Reply:
thanks for bringing this up @aduh95, this doesn't look like a blocker to me, I'll proceed with the release in order to not miss the scheduled date but it might be a good idea to float a patch / open an issue to fix this metadata right after this is out 😊